airbyte-integrations/connectors/source-stripe/source_stripe/manifest.yaml

```diff
@@ -0,0 +1,232 @@
+version: 6.33.0
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - events
+
+definitions:
+  # base components
+  base_stream:
+    type: DeclarativeStream
+    primary_key:
+      - id
+    retriever:
+      $ref: "#/definitions/base_retriever"
+    schema_loader:
+      type: JsonFileSchemaLoader
+      file_path: "./source_stripe/schemas/{{ parameters['name'] }}.json"
+
+  base_retriever:
+    type: SimpleRetriever
+    requester:
+      $ref: "#/definitions/base_requester"
+    record_selector:
+      type: RecordSelector
+      extractor:
+        type: DpathExtractor
+        field_path:
+          - data
+      schema_normalization: Default
+    paginator:
+      $ref: "#/definitions/base_paginator"
+
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.stripe.com/v1
+    authenticator:
+      $ref: "#/definitions/bearer_authenticator"
+    http_method: GET
+    request_headers:
+      Stripe-Version: "2022-11-15"
+      Stripe-Account: '{{ config.get("account_id") }}'
+
+  bearer_authenticator:
+    type: BearerAuthenticator
+    api_token: "{{ config['client_secret'] }}"
+
+  base_paginator:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: starting_after
+    page_size_option:
+      type: RequestOption
+      field_name: limit
+      inject_into: request_parameter
+    pagination_strategy:
+      type: CursorPagination
+      page_size: 100
+      cursor_value: '{{ response["data"][-1]["id"] }}'
+      stop_condition: '{{ not response.get("has_more", False) }}'
+
+  base_incremental_sync:
+    type: DatetimeBasedCursor
+    cursor_datetime_formats:
+      - "%s"
+    datetime_format: "%s"
+    step: P{{ config.get('slice_range', 365) }}D
+    cursor_granularity: P1D
```

Comment:
I'm surprised to see the granularity being "1 day" here. Should it be "1 second" because Stripe datetime are in epoch?

Reply:
Updated to PT1S